src/app/store.ts

```diff
@@ -1,10 +1,19 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
-// eslint-disable-next-line import/no-cycle
 import counterReducer from '../features/counter/counterSlice';
+import usersReducer from '../features/users/usersSlice';
+import postsReducer from '../features/posts/postsSlice';
+import postCommentsReducer from '../features/comments/сommentsSlice';
+import selectedPostReducer from '../features/selectedPost/selectedPostSlice';
+import authorReducer from '../features/author/autthorSlice';
```

Comment:
There is a typo in the import path for `authorReducer`. The directory name `autthorSlice` seems to be misspelled. Ensure that the path is correct to avoid import errors.